packages/compartment-mapper/src/bundle.js

```diff
@@ -245,7 +245,13 @@ export const makeBundle = async (read, moduleLocation, options) => {
   for (const module of modules) {
     module.indexedImports = Object.fromEntries(
       Object.entries(module.resolvedImports).map(([importSpecifier, key]) => {
-        key = aliases.get(key) ?? key;
+        // TODO Change to the following line, once our tools don't choke on `?.`.
+        // See https://github.com/endojs/endo/issues/1514
```

Comment:
that issue is closed. What was choking this time? We should have a new issue or at least re-open that one.

Reply:
could it be this? https://github.com/terser/terser/blob/master/CHANGELOG.md#v5193

Endo is on terser 5.16.6 presently